README.md

```diff
@@ -66,6 +81,8 @@ How does module.exports help us with separation of concerns?
 
 ```text
 
+module.exports allows elements of one .js file to be accessed in another .js file.  For example, one file could have "module.exports = mongoose.model("modelName")".  Another file can "require" mongoose and this modelname is available to use in the requiring file.  This allows for separate files to create a database and connecting it to mongoose.
+
```

Comment:
:+1: True, although this is not restricted to linking an application with a database. The important thing here is that we can contain related code in a single file.